filters.js

```diff
@@ -0,0 +1,36 @@
+export const mappedNetflix = (arr) => {
+    
+  return arr.map((item) => {
+    return {
+      "title": item.title,
+      "year": item.release_year,
+      "cast": item.cast,
+      "category": item.type,
+      "country": item.country,
+      "id": item.show_id
+    }
+    })
+}
+
+export const filterMoviesOnYear = (year, movies) => {
+  if (year) {
+    return movies.filter((item) => item.release_year === +year)
+  }
+
+  return movies;
+}
+
+export const filterMoviesByCountry = (country, movieList) => {
+  if (country) {
+    return movieList.filter((item) => item.country.toLowerCase().includes(country.toLowerCase()))
+  }
+
+  return movieList
+}
+
+export const filterMoviesByActor = (cast, movieList) => {
+  if (cast) {
+    return movieList.filter((item) => item.cast.includes(cast))
+  }
+  return movieList
+}
```

Comment:
All three functions are doing the same, you can make it one function and just accept different arguments :)